src/app_service/service/accounts/service.nim

```diff
@@ -377,20 +377,7 @@ QtObject:
       "UDPPort": WAKU_V2_PORT
     }
 
-    result["WalletConfig"] = %* {
-      "LoadAllTransfers": true,
-      "OpenseaAPIKey": OPENSEA_API_KEY_RESOLVED,
-      "InfuraAPIKey": INFURA_TOKEN_RESOLVED,
-      "InfuraAPIKeySecret": INFURA_TOKEN_SECRET_RESOLVED,
-      "AlchemyAPIKeys": %* {
-        "1": ALCHEMY_ETHEREUM_MAINNET_TOKEN_RESOLVED,
-        "5": ALCHEMY_ETHEREUM_GOERLI_TOKEN_RESOLVED,
-        "42161": ALCHEMY_ARBITRUM_MAINNET_TOKEN_RESOLVED,
-        "421613": ALCHEMY_ARBITRUM_GOERLI_TOKEN_RESOLVED,
-        "10": ALCHEMY_OPTIMISM_MAINNET_TOKEN_RESOLVED,
-        "420": ALCHEMY_OPTIMISM_GOERLI_TOKEN_RESOLVED
-      }
-    }
+    result["WalletConfig"] = NODE_CONFIG["WalletConfig"]
```

Comment:
@IvanBelyakoff the only caveat would be (but basically that states for the entire config object made for the login purpose): "don't add properties to the login node config that can be changed from within the app".

Could you add that comment to the top of the `getLoginNodeConfig` proc?

But actually, that comment should be added to the `network_constants.NODE_CONFIG` object for the `WalletConfig` property.